### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by yalh76**
## Solution
- *Upgrade to 1.21.0*
- *Name change*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
* An automatic package_check will be launch at https://ci-apps-dev.yunohost.org/, when you add a specific comment to your Pull Request: "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!"*

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: manifest.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-    "name": "Bitwarden",
-    "id": "bitwarden",
+    "name": "vaultwarden",
```
Comment (kay0u) [inline manifest.json:2]:
```suggestion
    "name": "Vaultwarden",
```

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
#Own CI result for Revision: 7b90e03c9cdb9b010fe9c3b208040e2aae91bc4a 

image

**Pull request merged**